CHANGELOG-LATEST.md

```diff
@@ -62,3 +63,8 @@ New types that wrap native types from Apple, Google and Amazon, and we cleaned u
 
 ### Moved classes
 - `Store` enum has been moved from `entitlement_info_wrapper.dart` to `store.dart`
+
+### Other
+* Support fastlane plugin revenuecat internal (#410) via Toni Rico (@tonidero)
+* Update pubspec.yaml (#406) via Corey Rabazinski (@CoreyRab)
+* Update bug_report.md (#402) via Joshua Liebowitz (@taquitos)
```

Comment:
Hmm I wonder if we should add these to the changelog, don't think they add much value... But well, it's NABD

Reply:
I wasn't sure either so if you thought the same, I will remove them haha